include/revng/StackAnalysis/StackAnalysis.h

```diff
@@ -52,4 +62,169 @@ class StackAnalysis : public llvm::ModulePass {
   std::string TextRepresentation;
 };
 
+struct FuncSummary {
+  FuncSummary(
+    model::FunctionType::Values Type,
+    std::set<llvm::GlobalVariable *> ClobberedRegisters,
+    llvm::SmallSet<std::pair<llvm::CallInst *, model::FunctionEdgeType::Values>,
+                   4> Result,
+    std::optional<uint64_t> ElectedFSO,
+    llvm::Function *FakeFunction = nullptr) :
```

Comment:
```suggestion
  FuncSummary(
    model::FunctionType::Values Type,
    const std::set<llvm::GlobalVariable *> &ClobberedRegisters,
    const llvm::SmallSet<std::pair<llvm::CallInst *, model::FunctionEdgeType::Values>,
                   4> &Result,
    std::optional<uint64_t> ElectedFSO,
    llvm::Function *FakeFunction) :
```

Reply:
We should review how `Result` is composed of, since the `CallInst`s I'm returning are all dead.